README.md

```diff
@@ -169,6 +169,9 @@ a Python version greater than 2.7 (Python 3.2 is not supported anymore).
 * **SciPy**: Import Matlab workspace files in the Variable Explorer.
 * **Cython**: Run Cython files in the IPython console.
 
+## Contributing
+
+Everyone is invited to contribute. [Read the contributing instructions](CONTRIBUTING.md), then get started.
```

Comment:
should I change this for the complete link? 
`https://github.com/spyder-ide/spyder/blob/master/CONTRIBUTING.md`